source/about/on-prem-skype-for-business-replacement.rst

```diff
@@ -0,0 +1,40 @@
+On-Premises Skype for Business Replacement
+===========================================
+
+As Skype for Business reaches its end-of-life, organizations with stringent security, compliance, and operational demands face a critical challenge: finding a secure, on-premises replacement that supports both air-gapped operations and enterprise-controlled external collaboration. For environments that cannot rely on cloud-based solutions, the need for a secure and flexible platform is paramount.
+
+Mattermost offers a purpose-built, on-premises collaboration platform designed to address these unique requirements. With robust support for private, air-gapped environments and tightly controlled external communications, Mattermost empowers organizations to maintain operational continuity while meeting the demands of high-security environments.
+
+Use cases
+-------------
+
+Air-gapped operations
+~~~~~~~~~~~~~~~~~~~~~
+
+Looking for a communication solution that eliminates reliance on external cloud providers while ensuring complete control over sensitive data and secure collaboration within their infrastructure? Mattermost provides a secure, on-premises platform that meets these needs, enabling organizations to maintain operational continuity and data sovereignty in fully disconnected environments.
+
+- **Secure Workflows Tailored for Air-Gapped Environments**: Enable private messaging, file sharing, and collaboration features like audio calls and screen sharing—all operating securely in fully air-gapped infrastructures.
+
+- **FIPS 140-2 Compliance**: Deploy securely with FIPS 140-2 validated images that comply with NIST 800-53 and FedRAMP standards, ensuring the platform is fit for handling classified or sensitive data.
```

Comment:
We can probably combine FIPS and STIG to one overview benefit. We should list the specific features similarly to how we did the other page - "Explore the possibilities"